server/config.ts

```diff
@@ -69,24 +69,24 @@ export default {
         response: Number(get('PRISON_API_TIMEOUT_RESPONSE', 10000)),
         deadline: Number(get('PRISON_API_TIMEOUT_DEADLINE', 10000)),
       },
-      agent: new AgentConfig(Number(get('PRISON_API_TIMEOUT_DEADLINE', 10000))),
+      agent: new AgentConfig(Number(get('PRISON_API_TIMEOUT_RESPONSE', 10000))),
     },
     prisonerSearch: {
       url: get('PRISONER_SEARCH_API_URL', 'http://localhost:8083', requiredInProduction),
       timeout: {
-        response: Number(get('PRISONER_SEARCH_TIMEOUT_RESPONSE', 10000)),
-        deadline: Number(get('PRISONER_SEARCH_TIMEOUT_DEADLINE', 15000)),
+        response: Number(get('PRISONER_SEARCH_TIMEOUT_RESPONSE', 15000)),
+        deadline: Number(get('PRISONER_SEARCH_TIMEOUT_DEADLINE', 10000)),
       },
-      agent: new AgentConfig(Number(get('PRISONER_SEARCH_TIMEOUT_DEADLINE', 15000))),
+      agent: new AgentConfig(Number(get('PRISONER_SEARCH_TIMEOUT_RESPONSE', 15000))),
     },
     restrictedPatientApi: {
       url: get('RESTRICTED_PATIENT_API_URL', 'http://localhost:8084', requiredInProduction),
       timeout: {
-        response: Number(get('RESTRICTED_PATIENT_API_TIMEOUT_RESPONSE', 10000)),
+        response: Number(get('RESTRICTED_PATIENT_API_TIMEOUT_RESPONSE', 30000)),
```

Comment:
turns out that it is the response, not the deadline that is causing the time out issues